### DISCUSSION TITLE
主线图设置一队全清，潜艇仅boss战+远洋支援的设置，远洋支援会在道中第一战就消耗掉

### DISCUSSION THREAD

**Discussion by Fe2O3444** [LmeSzinc/AzurLaneAutoScript#4474] [Q&A]
设置如图，是我哪里没弄对吗
![捕获](https://github.com/user-attachments/assets/7c2fce03-30fd-42fb-b600-201cfa966504)

**Comment by suzu0bronya** [comment]
将需要打的主线图中潜艇职能调整为待机
